api/src/policies/travel-authorizations-policy.ts

```diff
@@ -121,6 +121,18 @@ export class TravelAuthorizationsPolicy extends PolicyFactory(TravelAuthorizatio
       ]
     }
 
+    // TODO: consider moving state based check to the service layer since its business logic?
+    if (this.record.status === TravelAuthorization.Statuses.APPROVED) {
+      // TODO: consider using actuals columns instead of current columns?
+      return [
+        "daysOffTravelStatus",
+        "dateBackToWork",
+        "travelDuration",
+        "tripType",
+        "wizardStepName",
+      ]
+    }
```

Comment:
@datajohnson Should I make new columns for these values to denote that they are the "actuals" vs. estimates?

Currently the data model was design around all data being "actuals" but some fields around travel times turn out to need customization to account for flight delays and cancellations and such.

Reply:
Deferring to part 2 of this PR since the change will be quite large.